doc/source/index.rst

```diff
@@ -4,23 +4,107 @@
 
 .. include:: include/global.rst
 
-Welcome to the documentation of the Python interface of igraph
-==============================================================
+.. currentmodule:: igraph
+
+.. raw:: html
+
+    <style type="text/css">
+    div.twocol {
+        padding-left: 0;
+        padding-right: 0;
+        display: flex;
+        gap: 20px;
+    }
+    
+    div.twocol > div {
+        flex-grow: 1;
+        padding: 0;
+        margin: 0;
+    }
+    </style>
+
+
+python-igraph |release|
+=======================
+|igraph| is a fast open source tool to manipulate, analyze, and plot graphs or networks written in C, and `python-igraph` is |igraph|'s interface for the Python programming language.
```

Comment:
I'd phrase it like "igraph is a fast open-source tool to this-and-that. It is primarily written in C and C++. `python-igraph` is..."

Otherwise it sounds like the graphs or networks are written in C, not igraph itself. (At least to me -- I'm not a native speaker so I might be wrong here).